user_docs/en/userGuide.md

```diff
@@ -1653,53 +1658,107 @@ When the confirmation dialog appears, select "restart now" or "restart later" if
 
 ##### Save configuration on exit {#GeneralSettingsSaveConfig}
 
-This option is a checkbox that, when checked, tells NVDA to automatically save the current configuration when you exit NVDA.
+| . {.hideHeaderRow} |.|

```

Comment:
In our original discussion we raised the potential of moving these tables to the bottom of each user guide section.
I think this is a good idea and more common in reference materials - most people want a basic description first, then tables for further reference.

https://github.com/nvaccess/nvda/pull/15902#issuecomment-1867093375

Could  you also update userGuideStandards.md to note this change of policy?